tests/phpunit/tests/option/option.php

```diff
@@ -312,4 +312,151 @@ public function data_option_autoloading() {
 			array( 'autoload_false', false, 'no' ),
 		);
 	}
+
+	/**
+	 * @ticket 22192
+	 *
+	 * @covers ::add_option
+	 */
+	public function test_add_option_with_value_of_false_should_store_false_in_the_cache() {
+		add_option( 'foo', false );
+		$a = wp_cache_get( 'alloptions', 'options' );
+		$this->assertFalse( $a['foo'] );
+	}
+
+	/**
+	 * @ticket 22192
+	 *
+	 * @covers ::add_option
+	 */
+	public function test_add_option_with_value_of_false_should_store_empty_string_in_the_database() {
+		add_option( 'foo', false );
+
+		// Delete cache to ensure we pull from the database.
+		wp_cache_delete( 'alloptions', 'options' );
+
+		$this->assertSame( '', get_option( 'foo' ) );
+	}
```

Comment:
What is the purpose of these tests? They seem unrelated to this PR, since the `add_option()` function is not even touched here.